STAR.Web/Content/Site.css

```diff
@@ -3,27 +3,96 @@
     padding-bottom: 20px;
 }
 
+/* This isn't necessary, bootstrap takes care of this.*/
 /* Set padding to keep content from hitting the edges */
-.body-content {
+/*.body-content {
```

Comment:
Ok, I wasn't sure. I don't want to delete something that is suppose to be
there or someone is really attached to.



Kind regards,

Christine Iammarino
e.  ciammarino1@gmail.com
p.  614.226.7643



On Mon, Aug 15, 2016 at 10:16 AM, ManifestMike <notifications@github.com>
wrote:

> In STAR.Web/Content/Site.css
> <https://github.com/ManifestMike/STAR/pull/11#discussion_r74767524>:
>
> >  /* Set padding to keep content from hitting the edges */
> > -.body-content {
> > +/*.body-content {
>
> It's OK to just remove this code instead of commenting. You can place a
> comment on it in github (like this one) if you feel you need to communicate
> the reason why.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManifestMike/STAR/pull/11/files/97ada6c44b6c6bd08d1139c8449be3c70b4772b8#r74767524>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMNcUjTKkAfNObPcE8gzRAW086shUUSHks5qgHSngaJpZM4JkYea>
> .
>